README.md

```diff
@@ -1,9 +1,8 @@
 
 # devopsdays-cli
 
-[![travis Status](https://travis-ci.org/devopsdays/devopsdays-cli.svg?branch=master)](https://travis-ci.org/devopsdays/devopsdays-cli) [![Build status](https://ci.appveyor.com/api/projects/status/u7pu7ins2csxngxu?svg=true)](https://ci.appveyor.com/project/DevOpsDays/devopsdays-cli)
- [![Go Report Card](https://goreportcard.com/badge/github.com/devopsdays/devopsdays-cli)](https://goreportcard.com/report/github.com/devopsdays/devopsdays-cli) [![GoDoc](https://godoc.org/github.com/devopsdays/devopsdays-cli?status.svg)](http://godoc.org/github.com/devopsdays/devopsdays-cli) [![GitHub release](https://img.shields.io/github/release/devopsdays/devopsdays-cli.svg)](https://github.com/devopsdays/devopsdays-cli/releases) [![MIT License](http://img.shields.io/badge/License-MIT-yellow.svg)](LICENSE) [![ZenHub](https://raw.githubusercontent.com/ZenHubIO/support/master/zenhub-badge.png)](https://app.zenhub.com/workspace/o/devopsdays/devopsdays-cli/)<br />
- [![Coveralls](https://img.shields.io/coveralls/devopsdays/devopsdays-cli.svg)]() [![BCH compliance](https://bettercodehub.com/edge/badge/devopsdays/devopsdays-cli?branch=master)](https://bettercodehub.com/results/devopsdays/devopsdays-cli) [![Ebert](https://ebertapp.io/github/devopsdays/devopsdays-cli.svg)](https://ebertapp.io/github/devopsdays/devopsdays-cli) [![codebeat badge](https://codebeat.co/badges/9acd2699-7397-45fb-8219-e0e5a68c8399)](https://codebeat.co/projects/github-com-devopsdays-devopsdays-cli-master) [![CodeFactor](https://www.codefactor.io/repository/github/devopsdays/devopsdays-cli/badge)](https://www.codefactor.io/repository/github/devopsdays/devopsdays-cli)
+ [![Go Report Card](https://goreportcard.com/badge/github.com/devopsdays/devopsdays-cli)](https://goreportcard.com/report/github.com/devopsdays/devopsdays-cli) [![GoDoc](https://godoc.org/github.com/devopsdays/devopsdays-cli?status.svg)](http://godoc.org/github.com/devopsdays/devopsdays-cli) [![GitHub release](https://img.shields.io/github/release/devopsdays/devopsdays-cli.svg)](https://github.com/devopsdays/devopsdays-cli/releases) [![MIT License](http://img.shields.io/badge/License-MIT-yellow.svg)](LICENSE)<br />
```

Comment:
The README is dynamically generated at release, so these changes will be over-written. Changes should be made instead to `README.e.md`

Reply:
Oh yeah! I forgot about that 👀 I know you mentioned it when we chatted. I'll sort 👍 Need to remind myself to move the tests back to using Ponyville too!